app/reducers/app/index.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {combineReducers} from 'redux';
+
+import build from './build';
```

Comment:
These don't really need their own files, but 0/5

Reply:
Yeah I can set them in the same file but wanted to keep the same structure